tests/infra/remote.py

```diff
@@ -574,85 +542,70 @@ def __init__(
             [sealed_secrets] if sealed_secrets else []
         )
 
-        # If the remote needs to verify the quote, only a subset of arguments are required
-        if self.verify_quote:
-            cmd += ["--start=verify"]
-
-            if not other_quote:
-                raise ValueError(
-                    "Quote should be specified when starting remote in verify mode"
-                )
-            if not other_quoted_data:
+        cmd = [
+            self.BIN,
+            f"--enclave-file={lib_path}",
+            f"--enclave-type={enclave_type}",
+            f"--node-address={host}:{node_port}",
+            f"--rpc-address={host}:{rpc_port}",
+            f"--public-rpc-address={host}:{rpc_port}",
+            f"--ledger-file={self.ledger_file_name}",
+            f"--node-cert-file={self.pem}",
+            f"--host-log-level={host_log_level}",
+            f"--raft-election-timeout-ms={election_timeout}",
+        ]
+
+        if sig_max_tx:
+            cmd += [f"--sig-max-tx={sig_max_tx}"]
+
+        if sig_max_ms:
+            cmd += [f"--sig-max-ms={sig_max_ms}"]
+
+        if memory_reserve_startup:
+            cmd += [f"--memory-reserve-startup={memory_reserve_startup}"]
+
+        if notify_server:
+            notify_server_host, *notify_server_port = notify_server.split(":")
```

Comment:
Why *port? In case there's ":" in the port name? I think this should be `host, port = server.split(":")`, and it should legitimately break if there's any number of ":" other than one.